src/diffusers/configuration_utils.py

```diff
@@ -245,6 +245,34 @@ def from_config(cls, config: Union[FrozenDict, Dict[str, Any]] = None, return_un
             deprecate("config-passed-as-path", "1.0.0", deprecation_message, standard_warn=False)
             config, kwargs = cls.load_config(pretrained_model_name_or_path=config, return_unused_kwargs=True, **kwargs)
 
+        # Handle old scheduler configs
```

Comment:
This will need to be robust and probably kept for a while unless we can find a way to mass update configs on the Hub. It's working with some scheduler configs already, FlowMatch vs Beta is detected with `shift` and `beta_schedule`, I've already found an edge case in SANA's config because we integrated those scheduler changes into DPM so it has `beta_schedule` and no `shift` (it was called `flow_shift` instead).